### DISCUSSION TITLE
Scattering of Thermal Photons

### DISCUSSION THREAD

**Discussion by nballering** [dullemond/radmc3d-2.0#27] [General]
Dear @dullemond,

I'm using radmc-3d to simulate ice absorption features from protoplanetary disks viewed edge-on. In the mid-IR, most of the flux is thermal emission from the inner disk that is subsequently scattered. You warn in the manual that this scenario will result in noisy images unless nphot_scat is huge, and indeed I have found this to be the case. You end this section of the manual with "Currently no elegant solution is found, but maybe there will in the future. Stay tuned…". I'm wondering if there has been any progress towards an elegant solution, or are huge values of nphot_scat still the only option?

Thank you,
Nick Ballering

**Comment by dullemond** [comment]
Hi Nick,
there are no new solutions. But here are a few tips:

1. If the opacity has strong forward scattering (g factor that is close to 1) at the wavelengths you are interested in, this gives particularly bad shot noise. Since forward scattering is almost like no scattering, it is best to "chop" this strong forward scattering. If you use the optool tool of Carsten Dominik (http://github.com/cdominik/optool) to create your opacities (highly recommended, fantastic tool!), then you can add a command line option "chop 10", which chops the forward scattering off the phase function for the inner 10 degrees. That helps reduce the noise.
2. If your model is axisymmetric, use 2D coordinates (r,theta), with theta including both the top half and bottom half of the disk, with with axial symmetry. Then you can still use scattering_mode == 5, even though you have axial symmetry and non-isotropic scattering. It will be slow (because it has to calculate the scattering source function for a whole set of angles at each scattering event), but it is better than using a 3D model.
3. If scattering is not too important, you can also resort to isotropic scattering, and reduce the scattering cross section by a factor of (1-g) to "compensate" for the isotropic scattering approximation. 
4. If scattering is not at all important, switch it off (add "noscat" on the command line)

Hope these tips help.

Best wishes,
Kees

**Reply by nballering** [reply]
Hi Kees,

Thank you for the tips! I had been chopping the forward scattering using your python version of bhmie. I was not aware of optool, but it looks fantastic indeed. I was using a fully 3D grid but will now try using a single phi cell with scattering_mode 5. Hopefully that will yield some speed up.

Best,
Nick
